lib/weather.rb

```diff
@@ -0,0 +1,23 @@
+class Weather
+  attr_reader :weather
+
+  CONDITION = {
+      1 => 'clear',
+      2 => 'rain',
+      3 => 'cloud',
+      4 => 'heavy rain',
+      5 => 'stormy'
+    }
+
+  def initialize
+    @weather = random_weather
+  end 
+
+  def random_number
+    rand(1..5)
```

Comment:
```suggestion
    [array, containing, weather, conditions].sample
```
would be another way to achieve this :)
https://www.rubyguides.com/2015/03/ruby-random/